python/grass/jupyter/interact_display.py

```diff
@@ -40,113 +145,68 @@ class InteractiveMap:
     >>> m.show()
     """
 
-    def __init__(self, width=400, height=400, use_region=False, saved_region=None):
+    def __init__(
+        self,
+        width=400,
+        height=400,
+        tiles="CartoDB positron",
+        use_region=False,
+        saved_region=None,
+    ):
         """Creates a blank folium map centered on g.region.
 
+        tiles parameter is passed directly to folium.Map() which supports several
+        built-in tilesets (including "OpenStreetMap", "Stamen Toner", "Stamen Terrain",
+        "Stamen Watercolor", "Mapbox Bright", "Mapbox Control Room", "CartoDB positron",
+        "CartoDB dark_matter") as well as custom tileset URL (i.e.
+        "http://{s}.yourtiles.com/{z}/{x}/{y}.png"). For more information, visit
+        folium documentation:
+        https://python-visualization.github.io/folium/modules.html
```

Comment:
I think you need to support API_key at least to support _Cloudmade and Mapbox custom tilesets_ (https://pypi.org/project/folium/0.1.4/). Perhaps this is for a different PR/issue.